installer/instmaterials/info_zh_hans.txt

```diff
@@ -0,0 +1,3 @@
+﻿该软件被定义为“文本编辑器（共同开发版本）”。
+您可以免费用于商业和非商业用途，但源代码的版权由各部分所有者保留。
+无法保证该软件在分发时的完整性。
```

Comment:
@EldersJavas Can you please check if this file is valid or not.
These lines are used in "Information" wizard page.

If I select Traditional Chinese on the installer, I see garbled characters.

![image](https://user-images.githubusercontent.com/1131125/106281082-19059980-6282-11eb-891e-b64106d23048.png)

On the other hand, If I select Simplified Chinese on the installer, I can see human-readable texts.

![image](https://user-images.githubusercontent.com/1131125/106281376-7e598a80-6282-11eb-9f30-0b2ba8ddc80d.png)

Reply:
This is indeed a problem.
When compiling locally, if I use UTF-8, select Simplified Chinese on the installer, it is still garbled.
So, I had to change its encoding to GBK.This is no way.

Reply:
I tried to use it on a computer in Traditional Chinese and it was displayed normally.

Reply:
@EldersJavas Thank you for the fix. I've confirmed that the garbled Traditional Chinese texts are fixed now.

![image](https://user-images.githubusercontent.com/1131125/106349956-94fcf180-6315-11eb-8906-674b1439f600.png)